packages/util/src/equal.ts

```diff
@@ -11,6 +17,12 @@ export function equalObjects(
   );
 }
 
+/**
+ * Checks whether the two items contain the same content.
```

Comment:
[req] I think this could benefit from a bit more context. The current description sounds as if this did the same as: `a == b`.